libs/wingsdk/src/fs/fs.ts

```diff
@@ -461,6 +610,37 @@ export class Util {
     fs.symlinkSync(target, path, type);
   }
 
+  /**
+   * Match files using the patterns the shell uses.
+   *
+   * Built with the great `glob` package, based on https://www.npmjs.com/package/glob
+
+   * @param pattern The pattern to match.
+   * @param options Glob options.
+   * @returns List of matching files.
+   */
+  public static glob(pattern: string, options: GlobOptions = {}): string[] {
+    return glob.sync(pattern, options);
+  }
```

Comment:
I feel like we should be careful about blanket-inheriting JavaScript APIs with large existing surface areas. I have nothing against popular JavaScript libraries from an implementation perspective (they're clearly battle tested!) but I imagine their configuration options have acquired a lot of cruft. Regex and globbing are important enough utilities that I reckon there's value designing them well. Can we leave this outside of the PR?

Reply:
I prefer not to punt this. I don't see us investing the time in the foreseeable future on designing this API, and I trust that `glob`, as you say, has been battle tested, so I am comfortable inheriting it.

Let me know if you are okay with this.

Reply:
OK, reduced the API to the minimum I think it needed.